From 7e1003c4bedf3edc89f775bd6d5021efeb3fdfa0 Mon Sep 17 00:00:00 2001
From: Meron Cohen <mecohen@habana.ai>
Date: Thu, 08 May 2025 18:34:17 +0300
Subject: [PATCH] [SW-227605] g2e: sensors: support new chip names

Add G2E variant chip names to sensor configuration files:
HL225E-pci-* (OAM) and HL288E-pci-* (AIC).

Change-Id: If561920996c297f3051d6455e2b5b15cff22a295
Signed-off-by: Meron Cohen <mecohen@habana.ai>
---

diff --git a/package/sensors/hl_sensors.conf b/package/sensors/hl_sensors.conf
index 7d6801d..2f39e8d 100644
--- a/package/sensors/hl_sensors.conf
+++ b/package/sensors/hl_sensors.conf
@@ -128,7 +128,7 @@
 label curr4 "Vcore VDD output current (IR)"
 label curr5 "LPDDR VDDR output current (IR)"
 
-chip "HL225-pci-*" "HL225B-pci-*" "HL225C-pci-*" "HL225D-pci-*"
+chip "HL225-pci-*" "HL225B-pci-*" "HL225C-pci-*" "HL225D-pci-*" "HL225E-pci-*"
 
 label temp1 "On Chip 0"
 label temp2 "On Chip 1"
@@ -438,7 +438,7 @@
 
 ignore power1
 
-chip "HL288-pci-*" "HL288B-pci-*" "HL288C-pci-*" "HL288D-pci-*"
+chip "HL288-pci-*" "HL288B-pci-*" "HL288C-pci-*" "HL288D-pci-*" "HL288E-pci-*"
 
 label temp1 "On Chip 0"
 label temp2 "On Chip 1"
diff --git a/package/sensors/hl_sensors_internal.conf b/package/sensors/hl_sensors_internal.conf
index 2f06dc3..9b387d5 100644
--- a/package/sensors/hl_sensors_internal.conf
+++ b/package/sensors/hl_sensors_internal.conf
@@ -128,7 +128,7 @@
 label curr4 "Vcore VDD output current (IR)"
 label curr5 "LPDDR VDDR output current (IR)"
 
-chip "HL225-pci-*" "HL225B-pci-*" "HL225C-pci-*" "HL225D-pci-*"
+chip "HL225-pci-*" "HL225B-pci-*" "HL225C-pci-*" "HL225D-pci-*" "HL225E-pci-*"
 
 label temp1 "On Chip 0"
 label temp2 "On Chip 1"
@@ -436,7 +436,7 @@
 
 ignore power1
 
-chip "HL288-pci-*" "HL288B-pci-*" "HL288C-pci-*" "HL288D-pci-*"
+chip "HL288-pci-*" "HL288B-pci-*" "HL288C-pci-*" "HL288D-pci-*" "HL288E-pci-*"
 
 label temp1 "On Chip 0"
 label temp2 "On Chip 1"
